transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -343,6 +359,33 @@ final ChannelFuture initAndRegister() {
 
     abstract void init(Channel channel) throws Exception;
 
+    Collection<ChannelInitializerExtension> getInitializerExtensions() {
+        if (disableExtensions) {
+            return ChannelInitializerExtensions.empty().extensions();
```

Comment:
nit: if replace this with `Collections.emptyList()`, then method ChannelInitializerExtensions.empty() may be removed